### PR TITLE
Make image path relative

### PR DESCRIPTION
**Pull request by ITopiaJesseGoerzen**
Ensures legibility when opening as a file rather than a webpage. I need this to generate pdfs from the site files on github workflows.

### CODE REVIEW

**Comment by ITopiaJesseGoerzen** [comment]
Fixes #20, #16.

**Comment by b-bittner** [comment]
@ITopiaJesseGoerzen 

thanks for your contribution to the project, your help is really welcome.
could you please resolve the merge-conflict, due to the newest changes in the code base. Thanks!

**Comment by ITopiaJesseGoerzen** [comment]
@b-bittner Resolved. Good to hear that this repo is being updated!

**Comment by b-bittner** [comment]
thanks, this looks good. If you don't mind, I would wait a few day for an answer from #16 @ssaraswati if this would also solve his problem.

**Comment by ITopiaJesseGoerzen** [comment]
It should fix his issue, but the reason I need it isn't really urgent either.

**Comment by ITopiaJesseGoerzen** [comment]
It's been a week. @ssaraswati is probably not going to pay attention to this pull request for some time, if ever. I'll be afk for 5 days and available on Tuesday 2023-01-17 if you need to get ahold of me again before merging.

**Comment by ITopiaJesseGoerzen** [comment]
I'm back from hiatus. Ask if you need fixes or questions answered.

**Comment by ITopiaJesseGoerzen** [comment]
This change would fix #25 .

File: kroki/plugin.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def _save_kroki_image_and_get_url(self, file_name, image_data, files):
         mkdocs_file = File(get_url, self._tmp_dir.name, self._output_dir, False)
         files.append(mkdocs_file)
 
-        return f'/{get_url}'
+        return {mkdocs_file.url_relative_to(page.file)}
```
Comment (mat813) [inline kroki/plugin.py:95]: Shouldn't this still keep get_url ? Like, if I have more than one diagram on my page ?

**Pull request merged**